pkgs/development/compilers/llvm/17/libcxx/default.nix

```
@@ -54,6 +54,14 @@ stdenv.mkDerivation rec {
       hash = "sha256-LNoPg1KCoP8RWxU/AzHR52f4Dww24I9BGQJedMhFxyQ=";
       relative = "libcxx";
     })
+  ] ++ lib.optionals (stdenv.isDarwin && lib.versionAtLeast stdenv.hostPlatform.darwinMinVersion "10.13") [
```

```suggestion
  ] ++ lib.optionals (stdenv.isDarwin && lib.versionAtLeast "10.13" stdenv.hostPlatform.darwinMinVersion) [
```

I think the args to versionAtLeast are swapped.

```console
nix-repl> lib.versionAtLeast "10.13" "10.12" 
true
nix-repl> lib.versionAtLeast "10.13" "10.13"                             
true
nix-repl> lib.versionAtLeast "10.13" "10.14" 
false
```